src/components/alert/alert.stories.js

```diff
@@ -0,0 +1,255 @@
+    },
+    buttonText: {
+      name: "button text (slot, for demo purposes)",
+      description: "",
+    },
+  },
+};
+
+export const Default = ({ closable, icon, title, size, type, content }) => html`
+  <vl-alert
+    ?data-vl-closable=${closable}
+    data-vl-icon=${icon}
+    data-vl-title=${title}
+    data-vl-size=${size}
+    data-vl-type=${type}
+  >
+    <p>${content}</p>
+  </vl-alert>
+`;
+
+export const Alert = ({ closable, icon, title, size, type, content }) => html`
+  <vl-alert
+    ?data-vl-closable=${closable}
+    data-vl-icon=${icon}
+    data-vl-title=${title}
+    data-vl-size=${size}
+    data-vl-type=${type}
+  >
+    <p>${content}</p>
+  </vl-alert>
+`;
+
+Alert.args = {
+  title: "Opgelet!",
+  type: ALERT_TYPE.ERROR,
+  icon: ALERT_ICON.WARNING,
+  content: "U heeft geen rechten om deze actie uit te voeren.",
+};
+
+export const Info = ({
+  closable,
+  icon,
+  title,
+  size,
+  type,
+  content,
+}) => html`<vl-alert
+  ?data-vl-closable=${closable}
+  data-vl-icon=${icon}
+  data-vl-title=${title}
+  data-vl-size=${size}
+  data-vl-type=${type}
+>
+  <p>${content}</p>
+</vl-alert>`;
+
+Info.args = {
+  title: "Info",
+  type: ALERT_TYPE.INFO,
+  icon: ALERT_ICON.INFO_CIRCLE,
+  content:
+    "Als u vaststelt dat er foute informatie over u in het bestand van de Centrale voor Kredieten aan Particulieren staat, dan kunt u een rechtzetting aanvragen.",
+};
+
+export const Success = ({
+  closable,
+  icon,
+  title,
+  size,
+  type,
+  content,
+}) => html`<vl-alert
+  ?data-vl-closable=${closable}
+  data-vl-icon=${icon}
+  data-vl-title=${title}
+  data-vl-size=${size}
+  data-vl-type=${type}
+>
+  <p>${content}</p>
+</vl-alert>`;
+
+Success.args = {
+  title: "Gelukt!",
+  type: ALERT_TYPE.SUCCESS,
+  icon: ALERT_ICON.CHECK,
+  content:
+    "We hebben uw melding goed ontvangen en nemen deze spoedig in behandeling.",
+};
+
+export const Warning = ({ closable, icon, title, size, type, content }) => html`
+  <vl-alert
+    ?data-vl-closable=${closable}
+    data-vl-icon=${icon}
+    data-vl-title=${title}
+    data-vl-size=${size}
+    data-vl-type=${type}
+  >
+    <p>${content}</p>
+  </vl-alert>
+`;
+
+Warning.args = {
+  title: "Technische storing",
+  type: ALERT_TYPE.WARNING,
+  icon: ALERT_ICON.WARNING,
+  content:
+    "Door een technische storing is dit loket tijdelijk niet beschikbaar.",
+};
+
+export const WithButton = ({
+  closable,
+  icon,
+  title,
+  size,
+  type,
+  buttonText,
+  content,
+}) => html`
+  <vl-alert
+    ?data-vl-closable=${closable}
+    data-vl-icon=${icon}
+    data-vl-title=${title}
+    data-vl-size=${size}
+    data-vl-type=${type}
+  >
+    <p>${content}</p>
+    <button slot="actions" is="vl-button">${buttonText}</button>
+  </vl-alert>
+`;
+
+export const WithTitleSlot = ({
+  closable,
+  icon,
+  title,
+  size,
+  type,
+  slotTitle,
+  content,
+}) => html`
+  <vl-alert
+    ?data-vl-closable=${closable}
+    data-vl-icon=${icon}
+    data-vl-title=${title}
```

Comment:
data-vl-title mag weg omdat je de title slot gebruikt, is verwarrend

Reply:
Done.